src/main/java/com/bina/varsim/tools/evaluation/SAMcompare.java

```diff
@@ -226,9 +229,10 @@ public void run(String[] args) {
                             }
 
                             if (!closeAln) {
-                                if (mappingQuality > mapqCutoff) {
+                                if (getIdentity(rec) <= identityThreshold && mappingQuality > mapqCutoff) {
```

Comment:
Shouldn't it be mappingQuality < mapqCutoff since you are targeting low mapq alignments (multi-mapped)?